.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "python.analysis.extraPaths": [
+        "./backend/charm/lib"
+    ],
+    "cmake.sourceDirectory": "/home/mz2/Developer/test_observer/frontend/linux"
```

Comment:
Yes, this was intentional since VS Code looked like it was now doing relative path updates. However, what I committed was the wrong state of the file :facepalm: